lib/iris/plot.py

```diff
@@ -311,14 +313,18 @@ def _check_contiguity_and_bounds(coord, data, abs_tol=1e-4, transpose=False):
             np.logical_and(mask_invert[:-1, ], diffs_along_y))
 
         # If discontinuity occurs but not masked, any grid will be created
-        #  incorrectly, so raise a warning
+        #  incorrectly, so raise a warning for the user
         if not_masked_at_discontinuity_along_x or \
                 not_masked_at_discontinuity_along_y:
-            raise ValueError('The bounds of the {} coordinate are not'
-                             ' contiguous and data is not masked where the'
-                             ' discontiguity occurs. Not able to create a'
-                             ' suitable mesh to give to'
-                             ' Matplotlib'.format(coord.name()))
+            discontiguity_warning = 'The bounds of the {} coordinate are ' \
+                                    'not contiguous and data is not masked ' \
+                                    'where the discontiguity occurs. The ' \
```

Comment:
Not a change you've made but now rereading through this intial sentence I see it doesn't make sense. We discontiguous twice: "The bounds of the {} coordinate are not contiguous ... where the discontiguity occurs.".

Reply:
Oh I see, this says "there is a discontiguity and the data is not masked where that discontiguity is"